ecosystem/Editor-and-tool-support.md

```diff
@@ -1,51 +1,111 @@
+# Editor and tool support
 
-## Editors
+The PureScript ecosystem contains a number of editor plugins and tools to help you build libraries and applications. The set of available tools is growing continuously, but this documentation contains popular, stable tools that you can use today.
 
-#### Atom
+At minimum, almost all projects will use:
 
-- [purescript-contrib/atom-language-purescript](https://github.com/purescript-contrib/atom-language-purescript) provides syntax highlighting
-- [nwolverson/atom-ide-purescript](https://github.com/nwolverson/atom-ide-purescript) provides build support,   REPL, and autocomplete etc. via [psc-ide](https://github.com/purescript/purescript/tree/master/psc-ide)
+- The [PureScript compiler](https://github.com/purescript/purescript)
+- The [Spago](https://github.com/purescript/spago) build tool and package manager
 
-#### Emacs
-Install these two packages and follow the instructions in the `psc-ide-emacs` README.
-- [purescript-mode](https://github.com/purescript-emacs/purescript-mode) Syntax Highlighting and indentation (adapted from haskell-mode)
-- [psc-ide-emacs](https://github.com/purescript-emacs/psc-ide-emacs) Emacs plugin exposing the compilers IDE support
+These tools can be installed via [NPM](https://www.npmjs.com), among other installation methods.
 
-PSCI support via comint:
+## Editor support
 
-- [psci-mode](https://github.com/purescript-emacs/emacs-psci) is a REPL minor mode
+The PureScript compiler includes an IDE server, [`psc-ide`](https://github.com/purescript/purescript/tree/master/psc-ide), to supply tooling for editors. This server has been used to implement a [PureScript language server](https://github.com/nwolverson/purescript-language-server), which implements the [Language Server Protocol](https://langserver.org) for PureScript. Major editor tooling either uses `psc-ide` or the language server.
```

Comment:
How about "Major editor tooling either uses `purs ide server` directly, or via the language server."?